pkg/packagekit/updates.jsx

```diff
@@ -744,24 +776,47 @@ class OsUpdates extends React.Component {
                     return <div className="spinner spinner-lg progress-main-view" />;
 
             case "available":
+                // when unregistered, hide the Apply buttons and show a warning
+                if (this.state.unregistered) {
+                    unregisteredWarning = (<div>
```

Comment:
Indeed it should. I fixed this and the other place for that, and verified that in the existing code it already looks that way.